### DISCUSSION TITLE
Inúteu... #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#518] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/inuteu-1/

**Comment by one-eyed-willy** [comment]
Pensei que o maior nome fosse o dele



![null](<https://i.postimg.cc/hGcBmGkB/Episodios-17.webp>)

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/inuteu-1/#comment_1087459723>):

> Pensei que o maior nome fosse o dele[![null](<https://camo.githubusercontent.com/4678c68cc7ce514bda9d435ec80473a3a5aa592931354c282d832ba121a310db/68747470733a2f2f692e706f7374696d672e63632f684763426d476b422f457069736f64696f732d31372e77656270>)](<https://camo.githubusercontent.com/4678c68cc7ce514bda9d435ec80473a3a5aa592931354c282d832ba121a310db/68747470733a2f2f692e706f7374696d672e63632f684763426d476b422f457069736f64696f732d31372e77656270>)

![null](<https://i.postimg.cc/JhD2GCnL/unnamed.jpg>)
